### ISSUE TITLE
Enhancement: Frontend

### ISSUE THREAD

**Issue by Suvrat1629** [Nexusrex18/lumine#10]
The overall frontend needs enhancement. The buttons need to be visible, add some padding overall and make changes so that the divisions on every page are visible properly and that the page does not like look like one chump divided by thin lines.

**Comment by Harshil-0707** [comment]
I will try to enhance frontend because I have worked in react and tailwind for good time you can assign me this issue I would like contribute to your project.

**Comment by Harshil-0707** [comment]
I will have to design UI after that I will code it , so it might take some time are you ok with it??

**Comment by Suvrat1629** [comment]
Yes it is ok but we need to UI to look clean and simplistic, so try not to change the theme a lot as that will increase your work too. Also, how long do you intend to take? Give us an update after u finish designing and start coding.

**Comment by Harshil-0707** [comment]
It might take 3 to 4 days for UI design because I have college project and I am also learing backend so it will take time.I will mail you design if you are satisfied with that design then I will start the coding.

**Comment by Suvrat1629** [comment]
Sure go ahead any queries or updates put it up in this chat. Best of luck! :)

**Comment by Harshil-0707** [comment]
Ok,And thank you:)

**Comment by Harshil-0707** [comment]
I started working on UI when I run your project in my local machine UI of
localhost website does not look like UI of your hosted website
Here take a look
[image: screenshot.png]

**Comment by Harshil-0707** [comment]
May be there is a might be a problem when you hosted your website, I saw a
failure at this url "
https://github.com/Nexusrex18/lumine/tree/main/frontend/.bolt".

**Comment by slashexx** [comment]
@Harshil-0707 One clarification, the current hosted frontend is in the project directory.

You can take reference from the nextjs project in the frontend directory as it contains the previously designed frontend. There was some issue with some internal module in that directory which is why I had to refactor and make a project folder. 

So you'll have to work on that, goodluck !

**Comment by Harshil-0707** [comment]
Thank you for the clarification.

**Comment by Harshil-0707** [comment]
Do you want me to commit all changes at once or single page at a time?

**Comment by slashexx** [comment]
> Do you want me to commit all changes at once or single page at a time?

Anyway you want is fine, I suggest you open a draft PR and slowly keep updating it at your pace if that's fine with you.

**Comment by Harshil-0707** [comment]
Sure,I will do as you suggested.

**Comment by Harshil-0707** [comment]
My work is done.
